### PR TITLE
updated according to the new travis-ci infrastructure

### PR DESCRIPTION
**Pull request by liuggio**
cfr http://docs.travis-ci.com/user/workers/container-based-infrastructure/

### CODE REVIEW

**Comment by localheinz** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: php
 
+sudo:false
```
Comment (localheinz) [inline .travis.yml]: @liuggio 

Maybe, just for a consistent style, add a space after the `:`?

Comment (liuggio) [inline .travis.yml]: ops sorry

**Comment by liuggio** [comment]
@localheinz @nicolopignatelli  fixed and stashed this important PR

**Comment by nicolopignatelli** [comment]
could you also please remove php 5.3 from the build matrix?

**Comment by nicolopignatelli** [comment]
solved by #66

**Pull request closed without merging**